code/_helpers/washing.dm

```diff
@@ -1,89 +1,9 @@
 /proc/wash_mob(mob/living/washing)
-
 	if(!istype(washing))
 		return
-
 	var/mob/living/L = washing
-
 	if(L.on_fire)
-		L.visible_message(SPAN_DANGER("A cloud of steam rises up as the water hits \the [L]!"))
+		L.visible_message("<span class='danger'>A cloud of steam rises up as the water hits \the [L]!</span>")
```

Comment:
why remove the span macro??

Reply:
also this should probably be a balloon alert instead

Reply:
No it shouldnt

Reply:
> No it shouldnt

buh buh my TG feedback guidelines